### PR TITLE
profile changes

### PR DESCRIPTION
**Pull request by Aarontama**

### CODE REVIEW

File: users/tamashia/index.md [hunk 1]
```diff
@@ -6,9 +6,80 @@ author: Aaron Tamashiro
 Aaron Tamashiro
 ================================
 
-tamashia@oregonstate.edu
-
 <img src="{{ site.url }}users/tamashia/images/ME.jpg" width="200">
 
-I am a Ph.D. candidate in Nuclear Engineering in the [School of NUclear Science and Engineering].
-I like mushroom pizza. 
+I am a first year M.S. and Ph.D. student in Nuclear Engineering in the [School of NUclear Science and Engineering](https://ne.oregonstate.edu) at [Oregon State University](https://oregonstate.edu). My major advisor is [Todd Palmer](https://rtrp.github.io/osu-transport/palmerts/).
```
Comment (albeanth) [inline users/tamashia/index.md]: ~~NUclear~~ Nuclear

File: users/tamashia/index.md [hunk 2]
```diff
@@ -6,9 +6,80 @@ author: Aaron Tamashiro
...
+
+***
+
+## Research
+
+* In collaboration with Oak Ridge National Laboratory (ORNL), the following tasks will be accomplished:
+  * Connect the multigroup transport option to the ORIGEN depletion capability within Shift
+  * In collaboration with ORNL staff, define a set of verification test problems and evaluation metrics.
+  *  Investigate the impact of energy group structure on the accuracy and efficiency of depletion calculations in comparison with results from a) Shift continuous energy depletion, b) SERPENT, and c) VESTA.
+  * Investigate the relative performance of various depletion time-stepping options using the multigroup Monte Carlo
```
Comment (albeanth) [inline users/tamashia/index.md]: what do you think of below?

"Investigate the relative performance of various ~depletion~ time-stepping options using ~the~ multigroup Monte Carlo depletion techniques"

File: users/tamashia/index.md [hunk 3]
```diff
@@ -6,9 +6,80 @@ author: Aaron Tamashiro
...
+  * Employ the multigroup Monte Carlo/depletion option as a predictor step with the continuous energy option at the corrector as a higher order predictor/corrector depletion approach. Evaluate accuracy and efficiency in comparison with the existing predictor/correction method that involves only the continuous energy Monte Carlo coupled with ORIGEN for depletion.
+
+***
+
+### Work and Research Experience
+
+* Graduate Research Assistant: September 2016 - Present
+  *Working on a research project with Oak Ridge National Laboratory on Multi-group Monte Carlo.
```
Comment (albeanth) [inline users/tamashia/index.md]: " Laboratory on ~Multi-group~ multigroup Monte Carlo."

or you can leave it as is and change the other instances of multigroup to multi-group. just for consistencies sake is all..

**Comment by Aarontama** [comment]
I think everything looks good now

**Comment by albeanth** [comment]
looks good.

**Pull request merged**